packages/wallet/src/services/DelegationTracker/DelegationTracker.ts

```diff
@@ -73,10 +73,9 @@ export const certificateTransactionsWithEpochs = (
 
 const hasDelegationCert = (certificates: Array<Cardano.Certificate> | undefined): boolean =>
   !!certificates &&
-  certificates.some((cert) =>
-    [...Cardano.RegAndDeregCertificateTypes, ...Cardano.StakeDelegationCertificateTypes].includes(
-      cert.__typename as Cardano.RegAndDeregCertificateTypes | Cardano.StakeDelegationCertificateTypes
-    )
+  certificates.some(
+    (cert) =>
+      !!Cardano.isCertType(cert, [...Cardano.RegAndDeregCertificateTypes, ...Cardano.StakeDelegationCertificateTypes])
```

Comment:
`!!` operator seems to be redundant: type checkers return `boolean`s.

Reply:
Done here: [5138707](https://github.com/input-output-hk/cardano-js-sdk/pull/1074/commits/5138707eaf274991bdc1231ba64e14b79c2ba4db)